src/content/h1/guides/map-making/kill-volumes/readme.md

```diff
@@ -1,14 +1,15 @@
-Map authors who want to prevent players from reaching certain areas can use **kill volumes** (aka **death barriers** or **kill zones**). These are cubic 3D trigger volumes which are scripted to instantly kill any player that enters them. Adding them to a map is easy:
+Map authors who want to prevent players from reaching certain areas can use **kill volumes** (aka **death barriers** or **kill zones**). These are cuboid 3D trigger volumes which are scripted to instantly kill any player that enters them. Adding them to a map is easy:
 
 # Adding trigger volumes
 The first step is to add trigger volumes to your map's [scenario][] using [Sapien][].
 
 1. Select "Trigger volumes" in the Hierarchy view
 2. Right click on the ground in the Game window to place a volume
 3. Left click and drag the faces of the new volume to adjust its size
-4. Using the Properties palette, give the volume a suitable name like `kill_volume` (should be lower-case)
+4. Rotate the volume using the active marker if needed
+5. Using the Properties palette, give the volume a suitable name like `kill_volume` (should be lower-case)
```

Comment:
Pretty sure markdown lets you use `1. xyz` for each item in an ordered list, so you can avoid needing to manually re-number list elements like this.

Reply:
Whoa, you're right! Nice tip